### PR TITLE
Populate vulnerability.reference with a link to NVD

### PR DESCRIPTION
**Pull request by maxcold**
### Summary of your changes

This PR moves the logic existing in Kibana to Cloudbeat of populating the NVD link instead of the Aquasec link in `vulnerability.reference` field.

### Screenshot/Data

the field should look like the following after the change
```
{
...
"reference": "https://nvd.nist.gov/vuln/detail/CVE-2022-3080",
...
}
```

### Related Issues

- Fixes: https://github.com/elastic/cloudbeat/issues/869#top

### Checklist
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -73,3 +73,4 @@ terraform.rc
 # Vulnerability management
 db/
 fanal/
+java-db/
```
Comment (maxcold) [inline .gitignore:76]: not sure if it's smth specific to my machine or not, but I had this untracked dir after running cloudbeat locally with vulnerabilities yaml

File: vulnerability/events_creator.go [hunk 2]
```diff
@@ -391,6 +391,20 @@ func (e EventsCreator) getCVSSScore(vul trivyTypes.DetectedVulnerability) float6
...
 }
 
+const NVDVulnDetailBaseURL = "https://nvd.nist.gov/vuln/detail/"
+
+func getReference(vul trivyTypes.DetectedVulnerability) string {
+	if vul.CVSS == nil {
+		return vul.PrimaryURL
+	}
+
+	if _, ok := vul.CVSS[trivyVul.NVD]; ok {
```
Comment (maxcold) [inline vulnerability/events_creator.go:401]: I'm assuming that if there is a NVD in CVSS then this vulnerability exists in NVD database and the link will be valid. It's a bit brittle in my opinion, but if we want to replace aquasec links that's the simplest way I could find

Comment (amirbenun) [reply vulnerability/events_creator.go:401]: Also, as far as I understand this is the same solution we have today in kibana, right?

Comment (maxcold) [reply vulnerability/events_creator.go:401]: yes, exactly!

Comment (amirbenun) [reply vulnerability/events_creator.go:401]: nit: invert the `if`
```
if _, ok := vul.CVSS[trivyVul.NVD]; !ok {
  return vul.PrimaryURL
}
```

**Review by amirbenun — APPROVED** [review]

**Comment by maxcold** [comment]
@amirbenun updated in case you want to take a look

**Pull request merged**